modules/db-backup-restore/restore/templates/job.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: restore
+spec:
+  template:
+    spec:
+      affinity: {{ .Values.affinity | toYaml | nindent 6 }}
+      serviceAccountName: sa-postgres-restore
+      containers:
+        - name: restore-postgresdb
+          image: divyanshujain11/rds:postgres-backup-restore
```

Comment:
@divyanshujainSquareops  Why are we not using SquareOps-owned registry to store and use docker images?